ClosedXML.Report/RangeInterpreter.cs

```diff
@@ -116,7 +116,15 @@ public virtual void EvaluateValues(IXLRange range, params Parameter[] pars)
                 }
                 catch (ParseException ex)
                 {
-                    Debug.WriteLine("Cell value evaluation exception (range '{1}'): {0}", ex.Message, range.RangeAddress);
+                    if (ex.Message == "Unknown identifier 'item'" && pars.Length == 0)
```

Comment:
Relying on a concrete exception message is a very fragile approach. What you need is to define another class of exception (e.g. `RangeParseException : ParseException`) and then to check `if (ex is RangeParseException ...)`.

Reply:
Also, it has to be covered with test

Reply:
ParseException is thrown from System.Linq.Dynamic and I can not control it. So I have to check the text of the message. I could not think up another solution.

Reply:
Ah, sorry, I thought it is thrown by `ClosedXML.Report`. In this case, it needs unit tests even more so if the exception message changed we'd know about it.